lib/hooks/endorsed/index.js

```diff
@@ -1,32 +1,21 @@
-const { get, merge } = require('lodash');
+const { get } = require('lodash');
+const project = require('./project');
 
 const {
   withLicensing,
   withInspectorate
 } = require('../../flow/status');
 
-const Messager = require('../../messager');
-
 module.exports = settings => {
 
-  const messager = Messager(settings);
-
   return model => {
     const type = get(model, 'data.model');
 
     switch (type) {
       case 'pil':
         return model.setStatus(withLicensing.id);
       case 'project':
-        return Promise.resolve()
-          .then(() => {
-            return messager({ ...model.data, action: 'submit-draft' });
-          })
-          .then(() => {
-            const data = merge(model.data, { meta: { authority: 'Yes' } });
-            return model.patch(data);
-          })
-          .then(() => model.setStatus(withInspectorate.id));
+        return project(settings)(model);
```

Comment:
Again, can we pull the `project(settings)` bit up into the outer function so we're not re-initialising all the things per invocation?

Reply:
ah yeah  - good spot